src/test/java/com/epam/tc/hw2/ex1/ExerciseFirstTest.java

```diff
@@ -0,0 +1,71 @@
+package com.epam.tc.hw2.ex1;
+
+import com.epam.tc.hw2.BrowserTestBase;
+import java.util.List;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+
+public class ExerciseFirstTest extends BrowserTestBase {
+    @Test
+    public void testIndexPage() {
+        // 5. Assert that there are 4 items on the header section are displayed and that they have proper texts
+        SoftAssertions softAssertions = new SoftAssertions();
+        List<WebElement> header = driver.findElements(By.cssSelector(".uui-navigation.nav.navbar-nav.m-l8 > li"));
+        softAssertions.assertThat(header.size()).as("check the size of header").isEqualTo(4);
+        List<String> headerItems = List.of("HOME", "CONTACT FORM", "SERVICE", "METALS & COLORS");
```

Comment:
Let's say expectedHeaderItems, for example. It would be easier to understand what's going on.